### PR TITLE
 NETOBSERV-1786: Add support to build file based catalog

### PR DESCRIPTION
**Pull request by OlivierCazade**
## Description

This catalog will replace the catalog we currently use, for now, we have Makefile target for both.

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -429,6 +437,14 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(OCI_BIN) push ${BUNDLE_IMAGE};
 
+.PHONY: file-based-catalog-build
```
Comment (msherif1234) [inline Makefile:440]: just curious why did  u added those two new targets since konflux pipeline will use the containerfile directly ?

Comment (OlivierCazade) [reply Makefile:440]: We currently have target to build a regular catalog, but if I understand correctly at some point this catalog will be replaced by file based catalog.

I wanted to introduce the new targets without replacing the old ones for now.

**Pull request merged**